assets/scss/admin/forms.scss

```diff
@@ -508,16 +508,6 @@ ASIDE
 				cursor: pointer;
 			}
 
-			.give-remove:hover {

```

Comment:
@raftaar1191 don't remove the hover effect. Just fix the hover so it's no jumpy.

Reply:
Oh I thought I have to remove that 

Okay fixing it again @DevinWalker